.wordlist.txt

```diff
@@ -670,7 +681,9 @@ subdirectory
 subexpression
 subfolder
 subfolders
+suboptimal
```

Comment:
: less than optimal : not at the best possible level. suboptimal performance. suboptimal conditions/results/solutions.